ibm_mq/tests/conftest.py

```diff
@@ -107,4 +107,4 @@ def spin_up_ibmmq():
         log_patterns=log_pattern,
         sleep=10
     ):
-        yield
+        yield common.INSTANCE
```

Comment:
You should be able to use the `instance` fixture.

Reply:
Can't because the scope of the dd_environment fixture is `session`, which is larger than the scope of the instance scope